datastore/__init__.py

```diff
@@ -211,19 +218,23 @@ async def upload_picture_unknown(cursor, user_id, picture_hash, container_client
         # Create picture instance in DB
         if picture_set_id is None :
             picture_set_id = user.get_default_picture_set(cursor, user_id)
-        
+            
+        folder_name = picture.get_picture_set_name(cursor, picture_set_id)
+        if folder_name is None:
```

Comment:
What happens if the user creates 2 folder without a name? Will this create an issue having 2 folder named General?

Reply:
Good point, that's a mistake. The folder name should only be General if it's the default picture_set or if the picture_set isn't specified. Else the folder name is the picture_set_id or his name.